modules/manage/partials/tiered-storage.adoc

```diff
@@ -582,7 +582,7 @@ NOTE: The `serviceAccount` annotations and the `statefulset` Pod labels are esse
 
 CAUTION: Do not set an object storage property to an empty string `""` or to `null` as a way to reset it to its default value.
 
-To configure access to ABS/ADLS with shared keys:
+To configure access to ABS/ADLS with account access keys:
```

Comment:
Just to verify: as far as I can tell, Azure account access keys grant [full access](https://learn.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&bc=%2Fazure%2Fstorage%2Fblobs%2Fbreadcrumb%2Ftoc.json&tabs=azure-portal#protect-your-access-keys) to storage account data. 

If so, then these instructions we provide for AWS S3 shared keys **don't** have an equivalent for Azure, correct?

> To configure access to an S3 bucket with access keys instead of an IAM role:
> 1. Grant a user the following permissions to read and create objects on the bucket to be used with the cluster (or on all buckets): GetObject, DeleteObject, PutObject, PutObjectTagging, ListBucket. 

Meaning there is no way to modify permissions if using account access keys, and if they want to go by "least privilege," they must use managed identities instead?

Reply:
SMEs--please provide this guidance ( @deniscoady @andijcr or @WillemKauf ) Thx.

Reply:
> Meaning there is no way to modify permissions if using account access keys, and if they want to go by "least privilege," they must use managed identities instead?

Good catch. yes, the granular permissions are only for azure managed identities